crates/aptos-ledger/src/lib.rs

```diff
@@ -203,12 +286,12 @@ pub fn get_public_key(display: bool) -> Result<String, AptosLedgerError> {
 /// # Arguments
 ///
 /// * `raw_txn` - the serialized raw transaction that need to be signed
-pub fn sign_txn(raw_txn: Vec<u8>) -> Result<Vec<u8>, AptosLedgerError> {
+pub fn sign_txn(raw_txn: Vec<u8>, path: &str) -> Result<Vec<u8>, AptosLedgerError> {
```

Comment:
nit / personal preference: I would swap the order of args here, `path` first and `raw_txn` after